docs/user-guide/connecting-widgets.md

```diff
@@ -49,11 +56,32 @@ In order to connect Charts, Tables or Counters widget with Maps widget, the proc
 
 When a pan or zoom operation is performed in the Map widget, the other connected widgets are spatially filtered according to the Map viewport.
 
-## Maps with Legends
+### Maps with Legends
 
 Also in this case the connecting procedure is similar to those seen previously, but now the information contained in the Legend widget doesn't change according with the map extension. An example can be the following:
 
 <img src="../img/connecting-widgets/legend-map.jpg" class="ms-docimage"/>
 
+## Connecting Table widgets with other widgets
+
+With the same procedure used for maps (see [previous section](connecting-widgets.md#maps-with-other-maps)) the user can connect Table widgets with:
+
+* Map widgets
+
+* Other tables widget, only if it refers to the same layer
+
+* Chart widgets, only if it refers to the same layer
+
+* Counter widgets, only if it refers to the same layer
+
+When a table is connected with other widgets, it became a *Parent Table* and a filter appears on the top.
+
+<img src="../img/connecting-widgets/table_filter.jpg" class="ms-docimage" style="max-width:700px;"/>  
+
+It is possible to apply a filter in the *Parent Table* simply by writing a text in the input field present at the top of each column:
+
+<img src="../img/connecting-widgets/filter_on_table.gif" class="ms-docimage" style="max-width:700px;"/>
 
+Once a widget is connected to a map that is in turn connected with a *Parent Table*, filtering the *Parent Table* will also filter all the other connected widgets accordingly:
```

Comment:
**Once a widget is connected to a map widget that is connected with a *Parent Table* at the same time, the filter applied in the *Parent Table* will also filter all the other connected widgets accordingly:**

_In questo caso è da descrivere cosa accade e quali sono i filtri applicati in base alla tipologia di connessione come indicato nel diagramma_ [qui](https://github.com/geosolutions-it/MapStore2/issues/4149#issue-489762940):

![image](https://user-images.githubusercontent.com/1280027/74720109-8832db00-5235-11ea-86b8-98826d392cc7.png)

Reply:
Si deve indicare meglio le tipologie di filtri che vengono applicati in base al caso come presente nel diagramma sopra riportato